src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
Inconsistencies are stupid. `int` is used internally by PHP, `int` is what's referenced by all major projects and should be standardized in league repos.
This argument is stupid. By this logic, none of the PSR standards are any concern at all. I mean, at the end of the day, they don't affect the function of the application do they?
Yeah, you wouldn't say that. Just stick with what people use, the fact is, `integer`, `boolean` etc. are all aliases of `int`, `bool` etc. We try to stay away from aliases since they are more likely deprecated.